streams-tests/jvm/src/test/scala/zio/stream/StreamSpec.scala

```diff
@@ -1628,27 +1628,27 @@ class StreamSpec(implicit ee: org.specs2.concurrent.ExecutionEnv) extends TestRu
   private def repeatedAndSpaced =
```

Comment:
Is there a possibility that this test is flaky? Can you take another look @regiskuckaertz ?

Reply:
Nevermind, bad rebase. Sorry.